src/main/scala/cromwell/engine/backend/local/LocalEngineFunctions.scala

```diff
@@ -15,7 +15,9 @@ class LocalEngineFunctions(executionContext: TaskExecutionContext) extends Engin
   override protected def read_int(params: Seq[Try[WdlValue]]): Try[WdlInteger] = {
     assertSingleArgument(params).map {
       case f: WdlFile =>
-        WdlInteger(io.Source.fromFile(f.value.toFile).mkString.toInt)
+        WdlInteger(io.Source.fromFile(f.value.toFile).mkString.trim.toInt)
```

Comment:
OK this is confusing.  It looks like `fromSource` carefully sets up a `BufferedSource` with a close function, which is duly registered, but then `close` is never called?  :confused: 

Do the ancients have a recommended alternative that doesn't leave streams open?